statsmodels/tsa/holtwinters.py

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul 12 09:35:35 2017
+
+@author: tvzyl
+"""
+import numpy as np
+import pandas as pd
+import warnings
+
+from collections import namedtuple
+from scipy.stats import boxcox
+try:
+    from scipy.special import inv_boxcox
+except ImportError:
+    inv_boxcox = lambda x, lam6da: (x**lam6da-1)/lam6da if lam6da != 0 else np.log(x)
```

Comment:
Yes it is the name of the parameter, I just use a "6" instead of a "b", maybe give it another name?